aiohttp/worker.py

```diff
@@ -158,6 +164,15 @@ def _create_ssl_context(cfg):
             ctx.set_ciphers(cfg.ciphers)
         return ctx
 
+    def _get_valid_log_format(self, source_format):
+        if source_format == self.DEFAULT_GUNICORN_LOG_FORMAT:
+            return self.DEFAULT_AIOHTTP_LOG_FORMAT
+        elif re.search(r'%\([^\)]+\)', source_format):
+            raise ValueError('Gunicorn style using `%(name)s` '
+                             'is not supported for the log formatting.')
```

Comment:
Please append  http://aiohttp.readthedocs.io/en/stable/logging.html#format-specification link to exception text.
Now exception states that something is wrong but doesn't point on solution.

But it's most likely configuration problem, not programming error.
The exception will be caught by admin or devops, not software developer.

I suggest to provide very detailed error info in these cases.